packages/hdwallet-core/src/bitcoin.ts

```diff
@@ -212,10 +213,171 @@ export interface BTCWalletInfo {
 }
 
 export interface BTCWallet extends BTCWalletInfo {
-  _supportsBTC: boolean;
```

Comment:
Let's add this back just in case (I removed since there were dupes, but I don't want to accidentally break logic around this)